### ISSUE TITLE
Any chance of adding support for multiple tag files at custom locations?

### ISSUE THREAD

**Issue by UrsaDK** [aki77/atom-autocomplete-ctags#23]
It would be awesome if in addition to the main tag file we could specify a list of additional tag files to load. Ideally, this list would be generated by a custom, file-type dependant command.

Let me explain this in a few more details: 

Many projects have dependencies which hardly change throughout the life of a project. These dependencies can have their tag files generated on install. Because dependencies don't change, we don't need to waist time regenerating their tags when we update the main tag file. Thus, this would massively speed up tag generation for projects with _a lot_ of dependencies.

Basically, we can already [generate tags on commit](http://tbaggery.com/2011/08/08/effortless-ctags-with-git.html) and [on  dependency install (in ruby)](https://github.com/tpope/gem-ctags), we can list all additional tags with `find $(bundle show --paths) -name tags -maxdepth 1 2>/dev/null`, but we have no way of using them. :(
